crates/astria-conductor/src/main.rs

```diff
@@ -0,0 +1,56 @@
+use std::process::ExitCode;
+
+use astria_conductor::{
+    conductor::Conductor,
+    config,
+};
+use tracing::{
+    error,
+    info,
+};
+
+// Following the BSD convention for failing to read config
+// See here: https://freedesktop.org/software/systemd/man/systemd.exec.html#Process%20Exit%20Codes
+const EX_CONFIG: u8 = 78;
```

Comment:
cool, are we going to add this to the other crates too?

Reply:
I could do this as part of https://github.com/astriaorg/astria/issues/398

Reply:
@steezeburger Awesome, sync with @dereksione who has been doing some config work in #330